### PR TITLE
fix: styling project management

### PR DESCRIPTION
**Pull request by mortada-codes**
This PR...

## Changes

- [x] Highlighted the active project in the table
- [x] Separated cluster management from project management  
- [x] project item hover color
- [x] display confirm message when delete

## Fixes

-

## How to test it

-

## Screenshots

- 

https://user-images.githubusercontent.com/20525304/150767308-7e7259d7-f312-4878-804d-2d3eea9badc2.mp4

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

**Comment by mortada-codes** [comment]
@tommywhitehead  we have inconsistent colors between project dropdown and cluster dropdown! 
more context https://kubeshop.slack.com/archives/C023JNPUSP8/p1643011938397600

File: src/index.css [hunk 1]
```diff
@@ -172,18 +172,25 @@ code {
...
 }
 
 .ant-table-body .ant-table-row:hover .projects-table-column-actions {
   display: flex;
 }
 
-.ant-table-body .ant-table-row .ant-table-cell:first-child {
-  border-left: 4px transparent solid;
+.ant-table-body .ant-table-row:hover {
+  background: #111d2c;
```
Comment (topliceanurazvan) [inline src/index.css:183]: Let's use the values from `Colors.ts`

File: src/index.css [hunk 2]
```diff
@@ -172,18 +172,25 @@ code {
...
+}
+
+.project-table-active-project td {
+  color: #13a8a8;
```
Comment (topliceanurazvan) [inline src/index.css:187]: Same as above.

File: src/index.css [hunk 3]
```diff
@@ -172,18 +172,25 @@ code {
...
+}
+
+.project-table-active-project > :first-child {
+  border-left: 4px #13a8a8 solid;
```
Comment (topliceanurazvan) [inline src/index.css:191]: Same as above.

File: src/index.css [hunk 4]
```diff
@@ -172,18 +172,25 @@ code {
...
 }
 
-.ant-table-body .ant-table-row:hover .ant-table-cell:first-child {
-  border-left: 4px #84e2d8 solid;
+.cluster-dropdown-item li:hover {
+  background: #111d2c;
```
Comment (topliceanurazvan) [inline src/index.css:195]: Same as above.

**Review by topliceanurazvan — CHANGES_REQUESTED** [review]
Left some comments, after fixing we can merge it! Great work! 👍🏻

**Review by topliceanurazvan — APPROVED** [review]

**Pull request merged**